docs/release-notes/latest.md

```diff
@@ -9,100 +9,101 @@ tags:
-- Logging into Semgrep Cloud Platform through the CLI associates your CLI user ID to your Semgrep Cloud Platform account. See the [<i class="fas fa-external-link fa-xs"></i> Anonymous User ID](https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md#anonymous-user-id) section for more details.
+- Fixed UI issues in the new onboarding flow.
+- Fixed an issue where Semgrep Cloud Platform could crash during the onboarding flow. <!--(#10940) -->
+- Various frontend fixes and improvements to the following:
+	- Finding detail page
+	- Projects page
+- Fixed an issue where the **Delete user** functionality did not work for some Semgrep orgs. <!-- (#10756) -->
 
 ### Changed
 
-- **SCM configuration:** Improved the **Delete message** when deleting SCMs, so that you are aware of the implications of removing an SCM. Many major Semgrep features rely on a connection with your source code manager, so take care when deleting SCMs.
+- Updated the default CircleCI YAML snippet to include full and diff scans. <!-- (#10678) -->
+
+## 💻 Semgrep Code
 
 ### Fixed
 
-- **GitLab:** Fixed the GitLab CI sample configuration file to help users onboard GitLab repositories more clearly. In particular, the configuration file now includes the `GITLAB_TOKEN` environment variable, which was previously only in the docs.
-- Fixed a timeout issue when syncing large numbers (15,000+) of GitHub repositories in Semgrep Cloud Platform.
-- Fixed performance issues when synchronizing Semgrep Cloud Platform Projects with their corresponding GitHub repositories <!-- 10156 -->
+- Speed and stability improvements across the product. Semgrep Code pages, such as Findings and Policies, now load faster.
+- **Semgrep Assistant:** Component tags are now visible for all Assistant users.
+    - **Component tags** use GPT-4 to categorize your code based on its function, such as:
+        - Payments
+        - User authentication
+        - Infrastructure
+    - By categorizing your code through component tags, Semgrep Assistant is able to help you prioritize **high-risk issues**, for example if Semgrep has detected a code finding related to payments or user authentication.
+    ![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
 
-## 💻 Semgrep Code
+## ⛓️ Semgrep Supply Chain
 
-### Changed
+### Added
 
-- **Findings page:** By default, the findings page now displays findings from **default (trunk or main) branches**. You can customize this filter by selecting a value from the **Branch** drop-down menu.
+- Added a new, public [<i class="fas fa-external-link fa-xs"></i> Semgrep Supply Chain API](https://semgrep.dev/api/v1/docs/#tag/SupplyChainService) where you can filter and query third-party vulnerability findings by a variety of parameters, such as:
+    - Severity
+    - Repository
+    - Exposure
+- **C# reachability** is now **GA (generally available)**. Semgrep Supply Chain has added reachability rule support for all C# CVEs from May 2022 onward.
```

Comment:
```suggestion
- **C# reachability** is now **GA (generally available)**. Semgrep Supply Chain has had reachability rule support for all C# CVEs since May 2022.
```

Reply:
Ah this one is tricky - we haven't "had" rule support since May 2022. Instead we just recently added those rules, working backward. Resolve if all good? :)